### PR TITLE
Implement TabOnlyHandler

### PR DESCRIPTION
**Pull request by danielemegna**
Implement `:tabonly` command as described in [vim tab tips page](https://vim.fandom.com/wiki/Using_tab_pages).

_____

My congrats to maintainers, I found this repo very tidy and code very easy to browse and understand. :clap: 

This pull request has been developed during [Open Source Saturday #23](https://www.meetup.com/it-IT/Open-Source-Saturday-Milano/events/259527969/) meetup in Milan.

### CODE REVIEW

File: src/com/maddyhome/idea/vim/ex/handler/TabOnlyHandler.kt [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import com.maddyhome.idea.vim.ex.CommandHandler
+import com.maddyhome.idea.vim.ex.CommandHandler.Flag.ARGUMENT_OPTIONAL
+import com.maddyhome.idea.vim.ex.CommandHandler.Flag.DONT_REOPEN
+import com.maddyhome.idea.vim.ex.ExCommand
+import com.maddyhome.idea.vim.ex.commands
+import com.maddyhome.idea.vim.ex.flags
+
+class TabOnlyHandler : CommandHandler(
+        commands("tabo[nly]"),
+        flags(ARGUMENT_OPTIONAL, DONT_REOPEN)
```
Comment (AlexPl292) [inline src/com/maddyhome/idea/vim/ex/handler/TabOnlyHandler.kt]: `ARGUMENT_OPTIONAL` - Does `tabonly` command take any argument? From the docs I can see that syntax of this command looks like `:{count}tabo[nly][!]` and there is no optional argument here. Or do I miss something?

And could you please add `RANGE_OPTIONAL` and `RANGE_IS_COUNT` flags? These flags are exactly for cases when command starts with `:{count}...` (even if you don't use this {count}).

**Comment by danielemegna** [comment]
Hello Alex! Thank you, I'm sorry for the delay.

You're absolutely right, I didn't take care of the flags.
Does it look right now?

File: src/com/maddyhome/idea/vim/ex/handler/TabOnlyHandler.kt (hunk 1 above)
Comment (danielemegna) [reply src/com/maddyhome/idea/vim/ex/handler/TabOnlyHandler.kt]: > :{count}tabo[nly][!]

Where did you see the `{count}` optional parameter? I cannot find it in the docs:

- http://vimdoc.sourceforge.net/htmldoc/tabpage.html
- https://vim.fandom.com/wiki/Using_tab_pages

I always use this command in vim without params anyway :-)

**Comment by danielemegna** [comment]
> :{count}tabo[nly][!]

Where did you see the `{count}` optional parameter? I cannot find it in the docs:

- http://vimdoc.sourceforge.net/htmldoc/tabpage.html
- https://vim.fandom.com/wiki/Using_tab_pages

I always use this command in vim without params anyway :-)

**Comment by AlexPl292** [comment]
> Where did you see the {count} optional parameter? I cannot find it in the docs:
Because you use the documentation for 7.3 version :)
https://github.com/vim/vim/blob/master/runtime/doc/tabpage.txt#L159

> Does it look right now?
In general, I meant that `ARGUMENT_OPTIONAL` should be replaced by `ARGUMENT_FORBIDDEN`. But, it seems, it is our problem that developers do not understand what flags should be put. I think I will merge your pull request and make the API more understandable.

**Comment by AlexPl292** [comment]
Related issue: https://youtrack.jetbrains.com/issue/VIM-1630

**Comment by danielemegna** [comment]
Awesome! Thank you Alex!

**Pull request merged**